float64 sx
float64 sy
float64 var_theta
bool walk_with_ball